0.0.1/milestone-1.md

```diff
@@ -0,0 +1,83 @@
+1. Alice shares that her info is alice.near
+1. Agent explains its ability to trade autonomously to realize gains as stablecoins: Your account balance on alice.near is 100 Near, with 1 Near currently priced at $3.18, your portfolio is worth $318.00. If your porfolio grows by 5%, you could swap $15 into stablecoins without reducing your portfolio below today's market value. Would you like me to set this as a swap goal?
+1. Alice says: No, what else can you do for me?
+1. Agent responds with the options: check your account details, check market prices, help you set an swap goal, recommend tokens to swap to realize your goals.
+1. Alice says: Ok, help me realize that weekly swap goal.
+1. Agent says: Ok great. I've saved a swap goal to realize $15 in USDC or USDT only if your portfolio grows by at least 5%, to at least reach $318.  Please confirm that this meets your expectations.
+1. Alice: yes.
+1. Agent says: In order to securely trade on your behalf I need your permission to transfer some tokens into a limited access trading account that you will own. It is from this trading account that I will be able to access your tokens and initiate transfers on your behalf. Shall we go ahead and create this limited access trading account, or would you like to know more details of how this works?
+1. Alice says: I'd like to know more.
+1. Agent says: Please dig into the technical deails @ <github link to beneivo labs auth_proxy readme> and post any questions to the Peerfolio team at <telegram dev support group TBD>
+1. Alice says: Ok I'm comfortable with that. Go head and setup my trading account.
+1. Agent: displays a Connect Wallet Button, which asks Alice to sign a message.
+1. Agent UI: on the callback from the signmessage action, the Agent says: Now let's determine the types and quantities of tokens to move into your trading account.  Peerfolio will only have access to the tokens on your trading account. All other tokens on alice.near will be untouchable. Would you like to fund your trading account with 10%, 15% or 20% of the 100 Near on your account? (Agent repeats the same quesiton for each supported token type: ETH, SOL, etc.)
+1. Alice: I'm ok with 10
+1. Agent UI: Ok understood. (Agent adds this trading % into state) In a moment you will be asked to approve two transactions. The 1st asks to create the trading account as a sub account of your main account called agent.alice.near with X Near transfered to your trading account. This trading account contains the logic that allows you to control where and how Peerfolio is allowed to trade on your behalf.  The second transaction asks to allow Peerfolio to request signatures to approve transactions on your trading account.
```

Comment:
I did some prototyping here and it doesn't seem like we are able to do request the user to sign transactions without us having at least a limited access key. This also shows up in our create_sub_account script where we need the parent account to sign the create-account and send-near transactions. 

Is there any other known way of requesting a user to sign a transaction (authless) I might be missing out on?

Reply:
To be clear this is what I'm referring to by having a limited access key
<img width="702" alt="image" src="https://github.com/user-attachments/assets/68511ab3-4a79-4a23-a69d-7c2c40b0b20f" />

![image](https://github.com/user-attachments/assets/8a06be84-7eec-4ed4-9df2-8dc0c3b61dfb)